src/common/threadpool.h

```diff
@@ -46,6 +46,9 @@ class threadpool
     static threadpool instance;
     return instance;
   }
+  static threadpool *getNew(unsigned max_threads = 0) {
```

Comment:
This function shouldn't exist. The point of the threadpool is to be a singleton that manages all threads, instead of having threads created willy-nilly everywhere, like the old code did.

Reply:
This is used just for the tests, which use different number of threads.